template/src/ontology/Makefile.jinja2

```diff
@@ -1556,6 +1556,28 @@ update_docs:
 	mkdocs gh-deploy --config-file ../../mkdocs.yaml
 {%- endif %}
 
+.PHONY: release_changelog
+release_changelog: $(REPORTDIR)/changelog.md $(REPORTDIR)/changelog.csv $(REPORTDIR)/changelog.yaml
+
+$(TMPDIR)/latest_release.obo: $(TMPDIR)
+	wget $(ONTBASE)/$(ONT)-base.owl -O $@.tmp.owl || { \
+		echo "Latest release couldn't be found, creating an empty OBO file instead"; \
+		echo "format-version: 1.2" > $@.tmp.owl; \
+	}
+	$(ROBOT) convert -i $@.tmp.owl --check false -f obo $(OBO_FORMAT_OPTIONS) -o $@.tmp.obo && grep -v ^owl-axioms $@.tmp.obo > $@ && rm $@.tmp.obo && rm $@.tmp.owl
```

Comment:
In ODK 1.6, with ROBOT 1.9.8 it is no longer necessary to grep out the `owl-axioms` tag, you can take the output of the `convert` command directly.

```suggestion
	$(ROBOT) convert -i $@.tmp.owl --check false -f obo $(OBO_FORMAT_OPTIONS) -o $@
```

Reply:
Removed the unnecessary step. Thanks for the heads-up